test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
If not, it can be added. It's (quality) packed into the BookDirectory field
so even though aren't walking offer directories, but rather the owner
directories shouldn't be a problem.